live-editing/configs/SelectConfigGenerator.ts

```diff
@@ -51,6 +53,17 @@ export class SelectConfigGenerator implements IConfigGenerator {
             shortenComponentPathBy: "/data-entries/select/"
         }));
 
+        configs.push(new Config({
+            additionalFiles: ["/src/app/data/heroData.ts"],
+            appModuleConfig: new AppModuleConfig({
+                imports: [IgxSelectModule, SelectSample5Component, IgxInputGroupModule],
+                ngDeclarations: [SelectSample5Component],
+                ngImports: [IgxSelectModule, IgxInputGroupModule]
```

Comment:
Not sure you need the `IgxInputGroupModule` here, as you're not using `igx-input-group` or `[igxInput]` in the component template